source/Playnite/GamesEditor.cs

```diff
@@ -467,6 +467,247 @@ public void OpenGameLocation(Game game)
             }
         }
 
+        public void UpdateGameSize(Game game, bool onlyIfDataMissing, bool updateGameOnLibrary, bool checkLastScanDate)
+        {
+            if (!game.IsInstalled)
+            {
+                return;
+            }
+
+            if (game.InstallSize != null && onlyIfDataMissing)
+            {
+                return;
+            }
+
+            ulong? scanSize;
+            try
+            {
+                scanSize = CalculateGameSize(game, checkLastScanDate);
+            }
+            catch (Exception e)
+            {
+                logger.Error(e, $"Failed to get InstallSize from game {game.Name} with install dir {game.InstallDirectory}");
+                throw e;

```

Comment:
Should be just `throw;` if you want to re-throw the same exception from catch block.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3017/commits/59263d4f192ed72e284ad362ccf4ddcc603c5302